layouts/partials/redirect.html

```diff
@@ -0,0 +1,12 @@
+{{ $action := .action }}
+{{ $action_style := $action.style | default "link" }}
+{{ $action_icon := $action.icon | default "arrow-left" }}
+{{ $action_icon_pos := $action.icon_position | default "left" }}
+{{ $page_url := trim $.URL "/" }}
+{{ $action_url := trim $action.url "/" }}
+<a href="{{ $action.url | absURL}}" target="_blank" rel="{{ if $action.new_window }}noopener {{ end }}{{ if $action.no_follow }}nofollow{{ end }}"
+    class="{{ if eq $page_url $action_url }} current-menu-item{{ end }} dropdown-item">
```

Comment:
Why are you inserting `current-menu-item` and `dropdown-item` classes here? We want it to be generic i.e. to work for any link (not just dropdown items)

Reply:
Ah yes I can remove the dropdown-item, that was just a remnant. current-menu-item is for the highlighting.

Reply:
I've spent a few hours on this, and right now the simplest way for this to work is to designate the redirect.html to be only for dropdown items, like it is in the action.html. They build their <a> and <span> there, and it seems that I will need to do it here as well. I've tried a few things, but building it in this file instead of a subcall seems to be the best so far.